### ISSUE TITLE
Changing password page should not move the navigation buttons upon toggling the keyboard

### ISSUE THREAD

**Issue by huayunh** [etn-ccis/blui-react-native-workflows#13]
#### Describe the bug
In changing the password page, the "cancel" and "update" buttons are pushed up when the keyboard is toggled.

#### What is the expected behavior?
Should be pinned at the bottom

#### What are the steps to reproduce?
1. yarn start:example-android
2. Login
3. Change password
4. Focus on the new password field
5. Observe that the "cancel" and "update" buttons are pushed up / pinned with the keyboard. 

#### Screenshots or links to minimum reproduction example
![image](https://user-images.githubusercontent.com/8997218/86382487-cae0d700-bc5b-11ea-8a15-603fb543a219.png)

#### What is your environment?
 Framework
RN

 Operating System
Android
		
#### Anything else to add?
From https://github.com/pxblue/react-native-workflows/pull/12

**Comment by joebochill** [comment]
I have spent the better part of two days trying to clean up the styles around this workflow and it is a giant can of worms. I got it to a point where it's starting to work really nice for iOS, but Android is terrible. The whole codebase is really messy and in need of a serious refactoring (creating reusable styles, wrapper components, etc.) - probably at least a week's worth of time. More work than I care to spend right now given our other priorities...I pushed up my work in progress to the [experiment/style-fixes](https://github.com/pxblue/react-native-workflows/tree/experiment/style-fixes) branch.
